lib/kubeclient.rb

```diff
@@ -8,9 +9,36 @@
 require 'kubeclient/replication_controller'
 require 'kubeclient/entity_list'
 require 'kubeclient/kube_exception'
-
+require 'kubeclient/watch'
 
 module Kubeclient
+
+  # HTTP Stream used to watch changes on entities
+  class WatchStream
```

Comment:
Yes, it is very common practice for one class per file, generally in a subdirectory structure that matches the underscorized full namespace.  Then, usually a top file requires it.

So, Kubeclient::WatchStream generally lives in lib/kubeclient/watch_stream.rb and in lib/kubeclient.rb you would `require 'kubeclient/watch_stream'`